### DISCUSSION TITLE
Converting after importing all images

### DISCUSSION THREAD

**Discussion by BJOHEI** [photoprism/photoprism#4465] [Q&A] [answered]
Trying here first, as it may be something I have missed
Is there a way to perhaps edit the docker-compose file so that PP imports all the xxxx amount of images from the import folder BEFORE it then starts converting using darktable_cli? I use PP mainly for the importing and folder sorting and on the slow laptop-"server" its taking a LONG time when each file is about 25-30MB in .cr3 format.
TIA

**Comment by graciousgrey** [comment, marked as answer]
You could disable the generation of preview images before importing the files.

https://docs.photoprism.app/user-guide/settings/library/#preview-images

If you later want them to be displayed in PhotoPrism you need to turn preview image generation back on and do a complete re-indexing.

**Reply by BJOHEI** [reply]
Not quite what I was hoping for, but if it works it works. Will have to try when I have new images to import.
Thanks for the tip
